src/commands/prune.ts

```diff
@@ -43,21 +50,29 @@ class Prune extends Command {
    */
   async run(): Promise<void> {
     const { flags: parsedFlags } = this.parse(Prune);
+    const isDryRun = parsedFlags['dry-run'];
     const config = await loadConfig();
     const connections = await resolveConnections(config, parsedFlags['connection-resolver']);
 
+    if (isDryRun) await printLine('• DRY RUN STARTED\n');
+
     const results = await prune(config, connections, {
```

Comment:
Maybe I don't understand but why do we need dry run logs here. We only need it at the top level? Just curious can you put a screenshot of migrate-rollback and prune as well. Thanks.

Reply:
![image](https://user-images.githubusercontent.com/13098867/110458965-7a781e00-80f4-11eb-88a7-c00b8be367e6.png)

cc @mesaugat

Reply:
Nice. Makes sense now. 👍